docs/getting-started-guides/windows/index.md

```diff
@@ -111,7 +84,7 @@ To run Windows Server Containers on Kubernetes, you'll need to set up both your
 
 
 1. Windows Server container host running the required Windows Server and Docker versions. Follow the setup instructions outlined by this help topic: https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/quick-start-windows-server.
-2. [Build](#Build) or download kubelet.exe, kube-proxy.exe, and kubectl.exe using instructions
+2. [Get Windows Binaries](#Get-Windows-Binaries) kubelet.exe, kube-proxy.exe, and kubectl.exe using instructions
```

Comment:
The `#link` needs to be all-lowercase to work.

Reply:
TIL